defaults/main.yml

```diff
@@ -142,6 +142,9 @@ consul_acl_master_token_display: "{{ lookup('env','CONSUL_ACL_MASTER_TOKEN_DISPL
 consul_acl_replication_enable: "{{ lookup('env','CONSUL_ACL_REPLICATION_ENABLE') | default('',true) }}"
 consul_acl_replication_token: "{{ lookup('env','CONSUL_ACL_REPLICATION_TOKEN') | default('', true) }}"
 
+## gossip encryption
+consul_encrypt_enable: "{{ lookup('env','CONSUL_TLS_ENABLE') | default(true, true) }}"
```

Comment:
Perhaps this Ansible variable can also have its own environment variable, like `CONSUL_ENCRYPT_ENABLE` for example instead of using `CONSUL_TLS_ENABLE`?